ansible/playbooks/tasks/azure-cluster-bootstrap.yaml

```diff
@@ -177,17 +235,17 @@
   changed_when: no
   when: ansible_hostname == 'vmhana01'
 
-- name: Reset crm maintainence facts
+- name: Reset crm maintenance facts
   ansible.builtin.set_fact:
-    crm_maintainence_mode: "{{ (crm_conf_show.stdout | regex_search('maintenance-mode=([a-z]*)', '\\1' ))[0] | default('unknown') }}"
+    crm_maintenance_mode: "{{ (crm_conf_show.stdout | regex_search('maintenance-mode=([a-z]*)', '\\1' ))[0] | default('unknown') }}"
     rsc_azure_events: "{{ crm_conf_show.stdout | regex_search('primitive rsc_azure-events') }}"
     cln_azure_events: "{{ crm_conf_show.stdout | regex_search('clone cln_azure-events') }}"
   when: ansible_hostname == 'vmhana01'
 
-- name: Ensure maintainence mode is deactivated
+- name: Ensure maintenance mode is deactivated
   ansible.builtin.command: crm configure property maintenance-mode=false
   when:
     - ansible_hostname == 'vmhana01'
-    - crm_maintainence_mode is true or crm_maintainence_mode == 'unknown'
+    - crm_maintenance_mode is true or crm_maintenance_mode == 'unknown'
 
 
```

Comment:
Coded added to skip certain steps depending on the fencing method used.  Also fixed the spelling of `maintenance ` in several vars :)